backends/python/server/text_embeddings_server/models/default_model.py

```diff
@@ -39,10 +39,9 @@ def embed(self, batch: PaddedBatch) -> List[Embedding]:
             kwargs["token_type_ids"] = batch.token_type_ids
         if self.has_position_ids:
             kwargs["position_ids"] = batch.position_ids
-
         output = self.model(**kwargs)
         embedding = output[0][:, 0]
-        cpu_results = embedding.view(-1).tolist()
+        cpu_results = embedding.reshape(-1).tolist()
```

Comment:
a view costs less than a reshape, why changing it ?

Reply:
Here if we send batched requests like:
`curl 127.0.0.1:8080/embed -X POST -d '{"inputs":["What is Deep Learning?", "It is a lovely day"]}' -H 'Content-Type: application/json'`
, it will return error `RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.`